vllm/v1/engine/core_client.py

```diff
@@ -156,15 +161,16 @@ def __init__(
     def shutdown(self):
         # Send shutdown signal to background process.
         self.should_shutdown = True
+        self.proc.join(5)
```

Comment:
tiny nit:
- it is possible that `self.proc` does not exist here
- Do we need to call `.join(5)` twice?